force-app/main/default/classes/Log.cls

```diff
@@ -128,6 +137,9 @@ public without sharing class Log {
                         .summary(e.getMessage())
                         .stackTrace(e.getStackTraceString())
                         .details(String.valueOf(e) + SPACE_SEP + e.getStackTraceString())
+                        .postProcessing(new PostProcessingControlsBuilder().setAll(true).build())
```

Comment:
@mikita-kniazeu can we leave out the post processing piece here? It's not set on any other .addError methods. By default it'll be true anyway